data/common.yaml

```diff
@@ -0,0 +1,28 @@
+rkhunter::allowhiddenfile:
+  - '/usr/lib*/hmaccalc/sha384hmac.hmac'
+  - '/usr/lib*/hmaccalc/sha512hmac.hmac'
+  - '/usr/sbin/.ispec.hmac'
+  - '/usr/sbin/.sshd.hmac'
+  - '/etc/.updated'
+  - '/usr/share/man/man1/..1.gz'
+  - '/usr/share/man/man5/.k5login.5.gz'
+  - '/usr/share/man/man5/.k5identity.5.gz'
+rkhunter::dbdir: '/var/lib/rkhunter/db'
```

Comment:
Was breaking up some of the array ones because it is easier read in yaml format for the arrays. The others were some suggested things by the rpm installation. I am removing them as they do not match to the defaults of rkhunter and are not a result of the system that it is installed on.